control-plane/README.md

```diff
@@ -16,6 +16,19 @@ The job control plane is used for controlling and monitoring aggregation jobs.
 * Run `firebase use --add` to add an alias. When prompted, select your **Project ID**, then give your Firebase project an alias. For more info, visit this [link](https://firebase.google.com/docs/cli#add_alias).
 * Run `yarn start` or `yarn deploy`
 
+## Making an Admin Account
+* After getting set up and deploying the JCP, go to the sign up page and create an account with the email "admin@chromium.org" and a password of your choice.
```

Comment:
We should allow the users to specify this account when they set up the control plane. I know this is hard coded for now, but could you create an issue for it?

Reply:
I created an issue